css/css-text/i18n/ja/css-text-line-break-ja-cj-loose.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html  lang="en" >
+<head>
+<meta charset="utf-8"/>
+<title>Line-break:loose, Conditional Japanese Starter (CJ) (ja)</title>
+<link rel="author" title="Richard Ishida" href="mailto:ishida@w3.org">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#line-break-property">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<meta name="assert" content="The browser allows a conditional Japanese starter at the beginning of a line.">
+<style type="text/css">
+@font-face {
+    font-family: 'mplus-1p-regular';
+    src: url('/fonts/CSSTest/mplus-1p-regular.woff') format('woff');
+    }
```

Comment:
I find it easier to read and and easier to spot errors when i indent my code as shown here (in Python style).

Reply:
Indeed, understood. But for me those unindented }s make it harder to see the clean lines of each set of related items. I dislike the common approach so much that i explicity change them all, since i use DreamWeaver and it automatically does what i don't want (and provides no way to adapt its behaviour that i can see).

Reply:
So this is a personal choice, but the tests are shared with all the browser vendors and the community.
I know there's no a coding style guide for WPT, but I haven't seen this kind of indentation
in the past, and it's not used in the rest of the css-text test suite.
So for me at least is quite confusing. :smile: